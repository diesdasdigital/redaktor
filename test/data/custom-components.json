{
  "default": {
    "required": {
      "title": "Components"
    },
    "fields": {},
    "components": [
      {
        "type": "custom-button",
        "data": {
          "label": "click me",
          "url": "https://diesdas.digital"
        }
      },
      {
        "type": "custom-image",
        "data": {
          "alt": "accessibility yo!",
          "url": "https://images.diesdas.digital/media/pages/global-assets/f5193acf57-1581604571/workshop-pass.svg?auto=format"
        }
      }
    ]
  }
}
