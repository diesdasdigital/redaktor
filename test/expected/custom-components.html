<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8" />
  <title>Components</title>
  <meta name="viewport" content="width=device-width,initial-scale=1" />
</head>
<body>
  <a href="https://diesdas.digital" class="custom-button">click me</a>
  <img src="https://images.diesdas.digital/media/pages/global-assets/f5193acf57-1581604571/workshop-pass.svg?auto=format" class="custom-image" alt="accessibility yo!">
</body>
</html>